_docs/bigquery/creating-bigquery-dataset.md

```diff
@@ -3,10 +3,26 @@ layout: page
 section: bigquery
 navigation_title: "Creating BigQuery dataset"
 title: "Creating BigQuery dataset"
-order: 4
-date: 2020-04-22
+order: 5
+date: 2020-04-29
 ---
 
 <!---
 In this article explained how to create Google BigQuery dataset
--->
\ No newline at end of file
+-->
+To store data in BigQuery you need to create a [dataset](https://cloud.google.com/bigquery/docs/datasets-intro){:target="_blank"}:
```

Comment:
Use texts and formatting from here - https://support.segmentstream.com/hc/en-us/articles/360013260880-Creating-Google-BigQuery-dataset